osu.Game/Screens/Ranking/ScorePanelList.cs

```diff
@@ -162,12 +172,15 @@ private void selectedScoreChanged(ValueChangedEvent<ScoreInfo> score)
             expandedTrackingComponent.Margin = new MarginPadding { Horizontal = expanded_panel_spacing };
             expandedPanel.State = PanelState.Expanded;
 
-            // Scroll to the new panel. This is done manually since we need:
-            // 1) To scroll after the scroll container's visible range is updated.
-            // 2) To account for the centre anchor/origins of panels.
-            // In the end, it's easier to compute the scroll position manually.
-            float scrollOffset = flow.GetPanelIndex(expandedPanel.Score) * (ScorePanel.CONTRACTED_WIDTH + panel_spacing);
-            scroll.ScrollTo(scrollOffset);
+            SchedulerAfterChildren.Add(() =>
```

Comment:
Kind of a scary addition. Everything seems fine so I'll let it pass, but is there a reason for this now being required?

Reply:
It allows for the fixes to the trigger value change flow that accompany it. Previously there was some dodge-as-fuck inline execution of the event in just the right way to make things work. Also panels were being expanded-then-contracted-then-expanded on load.

The reasoning for this being required is because at the point in time when this is run, the flow hasn't updated its alive children yet, so while the index (and offset) will be correct the `ScrollTo` gets limited to the scroll extent (500) instead of the actual panel location.